Automattic/wp-calypso
# Try/typescript plugin selectors

#### Proposed Changes

*

#### Testing Instructions



For every page be sure to also visit the production page and verify that everything looks identical. This PR is not supposed to change any functionality.

Check that the following works:
Plugin details page (/plugins/:plugin_slug)
* Activating/deactivating
* Enabling/disabling autoupdate
* Updating

Site details page (/plugins/manage/:site_slug)
* Activating/deactivating via toggle
* Enabling/disabling autoupdate via toggle
* Updating all plugins via primary update button
* Updating one plugin via update link
* Activating/deactivating via "edit all"
* Enabling/disabling autoupdate via "edit all"
* Updating via "edit all"
* Removing via "edit all"
* Removing via ellipsis

Plugins manage page
Site details page (/plugins/manag)
* Activating/deactivating via toggle
* Enabling/disabling autoupdate via toggle
* Updating all plugins via primary update button
* Updating one plugin via update link
* Activating/deactivating via "edit all"
* Enabling/disabling autoupdate via "edit all"
* Updating via "edit all"
* Removing via "edit all"
* Removing via ellipsis

Regression test the following:
* Jetpack plugin updates guided tour
* Marketplace
  * Checkout thank you
  * Plugin install
* Site settings (google analytics, cloudflare analytics, seo

#### Pre-merge Checklist





Related to #
